docs/Readme.md

```diff
@@ -31,7 +31,7 @@ NOTE: Depending on the region you choose to deploy the stack in - the deployment
 ## Stack Architecture
 
 This template set deploys the following infrastructure core to your LAMP instance:
-- Autoscaling web frontend layer (Nginx for https termination, Varnish for caching, Apache/php or nginx/php-fpm)
+- Autoscaling web frontend layer (Nginx for https termination, Apache/php or nginx/php-fpm)
```

Comment:
Has Varnish been removed? If so why? It was added in for a specific reason (though I do not recall why). Has load testing been done against this configuration?